client/src/Components/Employees/reducers.js

```diff
@@ -17,7 +17,6 @@ const employeesReducer = (state = initialState, action) => {
       return {
         ...state,
         loading: false,
-        error: false,
         employees: action.employees
```

Comment:
I mentioned changing this over to payload above. Would need to change it here too.
Payload is the Redux naming standard.

Reply:
Changed later